### DISCUSSION TITLE
word level timestamps

### DISCUSSION THREAD

**Discussion by li-henan** [openai/whisper#1497] [Q&A]
Dear author, could you help me to solve this problem?
I input :whisper p226_007.wav --model tiny --device cpu --word_timestamps True , cause I want to see word level timestamps, but errors occur like this : AssertionError: 3D tensors expect 2 values for padding , is my python version wrong?
Thanks very much for replying

**Comment by phineas-pta** [comment]
try other audio

**Reply by li-henan** [reply]
thanks for replying, what kind of audio should I input?Is there any requirements？

**Reply by phineas-pta** [reply]
your audio may be corrupted

try any other one

**Reply by li-henan** [reply]
but these wavs can be used in whisper p226_007.wav --model tiny --device cpu ，when not have --word_timestamps True
